aries_cloudagent/wallet/indy.py

```diff
@@ -705,6 +705,8 @@ async def set_did_endpoint(
         endpoint: str,
```

Comment:
You'll need to make these same changes in the askar.py version of the ledger

(this is enabled when you use `--wallet-type askar`)